### PR TITLE
[wip] App deployable from docker compose

### PR DESCRIPTION
**Pull request by scottPlusPlus**
Not ready to merge.

Dockerfile and docker-compose to deploy app via docker

Does NOT yet handle keycloak, but just wanted to get visibility on this.

### CODE REVIEW

**Comment by mikefranze** [comment]
On the conflicts with npm packages, the current main branch package.json and package-lock.json should be up to date. 
For environmental variables I had to set DEV_DATABASE=FALSE due to an SSL issue. 
@scottPlusPlus, was there anything else outside of keycloak left to add to this? @evanstucker-hates-2fa is going to work on reducing the docker image size and other optimizations.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+FROM node:16
```
Comment (mikefranze) [inline Dockerfile:1]: Read that node:16-alpine could reduce image size

**Comment by evanstucker-hates-2fa** [comment]
FYI, I'm working this in another branch in the star-server repo here: https://github.com/Equal-Vote/star-server/tree/238-reduce-size-of-container-image.

**Comment by ArendPeter** [comment]
We got the docker stuff worked out :), closing

**Pull request closed without merging**